### PR TITLE
Audio muted test

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: spot-webdriver/user/spot-session.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        meetingInput.submitMeetingName(testMeetingName);
+
+        const meetingPage = this.spotTV.getMeetingPage();
+
+        meetingPage.waitForVisible();
+
+        expect(meetingPage.getMeetingName()).toBe(testMeetingName);
+    }
+}
+
```
Comment (virtuacoplenny) [inline spot-webdriver/user/spot-session.js]: Do you think remotes and TVs should be obtained through the session?

File: spot-webdriver/user/spot-session.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        const testMeetingName = meetingName ? meetingName : `ui-test-${Date.now()}`;
+        const meetingInput = remoteControlPage.getMeetingInput();
+
+        meetingInput.submitMeetingName(testMeetingName);
+
+        const meetingPage = this.spotTV.getMeetingPage();
+
+        meetingPage.waitForVisible();
+
+        expect(meetingPage.getMeetingName()).toBe(testMeetingName);
```
Comment (virtuacoplenny) [inline spot-webdriver/user/spot-session.js]: I'm not sure how I feel about a jasmine expectation being defined in session. I'm not against it but I'm not used expects being located outside the spec file.

Comment (paweldomas) [reply spot-webdriver/user/spot-session.js]: I will look into that

File: spot-webdriver/specs/in-meeting.spec.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    meetingRemoteControlPage.waitForVisible();
+
+    it('can toggle audio mute', () => {
+        meetingRemoteControlPage.muteAudio();
+
+        meetingRemoteControlPage.waitForAudioMutedStateToBe(true);
+
+        meetingRemoteControlPage.unmuteAudio();
+
+        meetingRemoteControlPage.waitForAudioMutedStateToBe(false);
```
Comment (virtuacoplenny) [inline spot-webdriver/specs/in-meeting.spec.js]: I wonder if waiting for mute state should be part of the unmute/mute action. The downside I see is tests aren't as explicit about what is being tested.

Comment (virtuacoplenny) [reply spot-webdriver/specs/in-meeting.spec.js]: This applies to anything I've already written too.

**Pull request merged**